packages/@lightningjs/ui-components/src/components/Row/Row.test.js

```diff
@@ -431,31 +431,31 @@ describe('Row', () => {
       done();
     });
 
-    it('should pass on screen items to onScreenEffect', done => {
+    it('should pass on screen items to onScreenEffect', async () => {
       row.w = 200;
+      row.clipping = true;
```

Comment:
This test needs clipping to pass as expected. I'm not sure why it was passing before...